kubernetes-response-engine/deployment/falco-config/falco_rules.local.yaml

```diff
@@ -0,0 +1,38 @@
+####################
+# Your custom rules!
+####################
```

Comment:
Is this rules file really related to the integration, or is it for some specific example of suspicious behavior? I wouldn't mix up the two, otherwise people will think the integration only is related to cryptomining.

Reply:
Good catch, is an specific example of suspicious behavior.

Thanks!